pymongo/synchronous/monitoring.py

```diff
@@ -1,1903 +0,0 @@
-# Copyright 2015-present MongoDB, Inc.
```

Comment:
How come this file doesn't show up as a rename like message.py does? Any way we can have git recognize it to minimize code churn?

I have the same question for many of the other files as well.

Reply:
Seems like it's just git being confused. Let me do some commit reworking, I think I can significantly improve it.

Reply:
I was wrong sadly, there isn't a good way to do this. Because we are essentially removing a lot of the `asynchronous` and `synchronous` files and putting a single version into the top-level `pymongo` package, it complicates the history a lot for git. Git doesn't track renames/moves really, just the changes, so a "rename" just means it had few enough changes along with the location move to qualify.